queries.md

```diff
@@ -4,76 +4,152 @@
 
 ### 1. All the companies whose name match 'Babelgum'. Retrieve only their `name` field.
 
-<!-- Your Code Goes Here -->
+Filter:{name:"Babelgum"}
+Project:{name:1, _id:0}
+Sort:
+Skip:
+Limit:
 
 ### 2. All the companies that have more than 5000 employees. Limit the search to 20 companies and sort them by **number of employees**.
 
-<!-- Your Code Goes Here -->
+Filter:{number_of_employees: {$gt: 5000}}
+Project:
+Sort:{number_of_employees: 1}
+Skip:
+Limit:20
 
 ### 3. All the companies founded between 2000 and 2005, both years included. Retrieve only the `name` and `founded_year` fields.
 
-<!-- Your Code Goes Here -->
+Filter:{$and: [{founded_year:{$gte: 2000}},{founded_year:{$lte: 2005}}]}
```

Comment:
here you can save yourself some keystrokes by chaining the query of the founded year:
```js
founded_year: {  $gte: 2000, $lte: 2005  }
```